sdk/spring/azure-spring-boot-starter-keyvault-certificates/README.md

```diff
@@ -100,27 +100,24 @@ This starter allows you to securely manage and tightly control your certificates
 
     ![Copy secrets](resource/copy-secrets.png)
 
-To use the starter for server side SSL, you will need to add the following to
-your `application.yml` (if the application is using Spring Cloud Config 
-Server for its configuration add it to the `bootstrap.yml` of the application)
-
+Add these items in your `application.yml`:
```

Comment:
Not this line but the above `Using a client id and client secret` section. 

We already have this doc https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/spring/azure-spring-boot-samples/create-sp-using-azure-cli.md, and I think the steps of creating a service principal via Azure Portal is common to most of our starters. How about we put these steps and screenshots in the same place?

Reply:
Issue created: https://github.com/Azure/azure-sdk-for-java/issues/20143
I'll do that in another PR.